### PR TITLE
Add Eric's San Diego file to Master

### LINKED ISSUE #9: Create a file for githubteacher San Diego, CA
- [x] Create a branch!
- [x] Add commits
- [ ] Open a Pull Request
- [ ] Collaborate
- [ ] Receive Approval 👍 
- [ ] Merge :shipit:

### PR DESCRIPTION
**Pull request by githubteacher**
closes #9

### CODE REVIEW

File: githubteacher-sandiego.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+#### Places to Go
+- Coronado
+- Balboa Park
+
+#### Places to Eat
+- City Tacos
+
+#### Places to Avoid
+- Sea World
```
Comment (KellyYoho) [inline githubteacher-sandiego.md:13]: What would Shamu say?

**Comment by githubteacher** [comment]
> What would Shamu say?

@KellyYoho "Let me be free" 🐳

**Comment by githubteacher** [comment]
@brianamarie Does this look good to :shipit: ?

**Comment by brianamarie** [comment]
Awesome! 🚢

**Pull request merged**